python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -98,13 +122,23 @@ def _computePsfImage(self, position=None):
     if not isinstance(position, Point2D) and position is not None:
         position = Point2D(position[0], position[1])
 
-    for single in self.singles:
-        if position is None:
-            psf = single.getPsf().computeImage()
-            psfs.append(psf)
-        else:
-            psf = single.getPsf().computeImage(position)
-            psfs.append(psf)
+    for band, single in enumerate(self.singles):
+        try:
+            if position is None:
+                psf = single.getPsf().computeImage()
+                psfs.append(psf)
+            else:
+                psf = single.getPsf().computeImage(position)
+                psfs.append(psf)
+        except InvalidParameterError:
```

Comment:
I'm a bit confused as to why the try block doesn't throw an IncompleteDataError itself if it's always due to a missing PSF model?

Reply:
The `InvalidParameterError` is thrown by the `Exposure.getPsf` method, which is defined in `afw`. I could just allow that to fail and catch it in `ScarletDeblendTask.deblend` but in general I like to isolate the code that causes a known issue in case an issue somewhere else arises later with the same error type.